### ISSUE TITLE
Exception in composer when using Typo3 v10 without Dashboard-sysext

### ISSUE THREAD

**Issue by subnoodle** [teaminmedias-pluswerk/ke_search#409]
Dear KE-Search-Delelopers,
besides that you da a really great job with this extensions, i stepped into a pitfall right now:

I tried to setup an Typo3 v10 instance, using KE-Search but not the new Typo3 dashboard extension.
It took a while since i figured out that this one is required by ke_search, but seemingly it's missing in ke-search's composer file (didn't counterchecker this).
Could you please add it into the composer file OR check, if dashboard is installed before using dashboard's widgets (which would be nicer of course)?

The error was this one - it's more exactly not composer, but typo3cms extension:setupactive (contained as after update script in composer file):

```
> php vendor/bin/typo3cms extension:setupactive
  [ Symfony\Component\DependencyInjection\Exception\RuntimeException ]
  Invalid service "dashboard.widget.ke_search_index_overview": class "TYPO3\CMS\Dashboard\Widgets\BarChartWidget" does not exist.

```
Thank you very much,
Thomas

**Comment by nischcodes** [comment]
+1

Same here. Had to manually rename the Services.yaml in the Configuration folder to ".Services.yaml" so that typo3 doesn't load this services

**Comment by JohnNDK** [comment]
Install the required extension: typo3/cms-dashboard

**Comment by nischcodes** [comment]
@JohnNDK this is a bit of a bummer, because this introduces us to new uncertainties this late in our project. Especially if we won’t introduce our editors and editors with admin rights to this new module without checking it first. But thanks for your constructive suggestions and your reaction to my comment...very well, feeling my contribution and myself very much welcomed, thanks a lot! 

quick note: It would help much, if the packagist package would tell that the 3.6.0 version requires the dashboard extension but thats a story for another time...

**Comment by christianbltr** [comment]
I'm sorry for the inconvenience. The requirement has not been introduced intentionally. I would prefer not to set typo3/cms-dashboard as a requirement but to check if it is installed before registering the widgets. 

Please consider downgrading to 3.5.0 until 3.6.1 is out if your deployment fails with 3.6.0. Please expect the fixed version at the end of this week.

**Comment by nischcodes** [comment]
@christianbltr thank you very much for your message. I will wait for the fixed version.

**Comment by christianbltr** [comment]
This has been fixed in ke_search 3.6.1 which has just been released.
